functions/_fzf_functions

```diff
@@ -1,7 +1,15 @@
 #! /usr/bin/env sh
```

Comment:
are some of these functions defined here used in multiple places?  As a contributor, I found it a bit confusing to need to define my fzf functions in one directory, and use them in another, hence my focus on the single fzf.zsh class.  Beyond grouping any and all functions into the /functions directory, is there another reason for putting the fzf functions in a directory other than /utilities/fzf?

Reply:
TLDR: That is a great critique. Current organization began with different concerns and I haven’t reevaluated my organization as my dotfiles have evolved. I don’t have immediate plans to reorganize but I’ve made a note to change to something more like your suggestion in the future.

Long version: my dotfiles organizational structure comes mostly from [Holman’s dotfiles](https://github.com/holman/dotfiles). His structure has a `functions` folder of which all scripts are automatically added to the path (see [this line](https://github.com/nathanshelly/.files/blob/master/zsh/zshrc.symlink#L55)). As I’ve added more functions I never really reconsidered where to put them so I defaulted to the functions folder. When I went to make this PR I already had `fzf` related functions in the `$DOTFILES/functions/_fzf_functions` file so I defaulted to putting the new `fzf-ignored-files-widget` in that file.